packages/suite/src/views/guide/GuideCategory.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { variables } from '@trezor/components';
+import { Header, Content, ViewWrapper, GuideSelectionItem } from '@guide-components';
+import { useActions, useSelector } from '@suite-hooks';
+import { GuideType } from '@suite-types/guide';
+import { Translation } from '@suite-components';
+import * as guideActions from '@suite-actions/guideActions';
+import { getItemTitle } from '@suite-utils/guide';
+
+const Section = styled.div`
+    margin-bottom: 20px;
+
+    &:not(:last-of-type) {
+        margin-bottom: 100px;
+    }
+`;
+
+const SectionHeading = styled.h3`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    padding: 8px 0 18px 0;
+`;
+
+const Items = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+`;
+
+const GuideCategory = () => {
+    const { item, language } = useSelector(state => ({
+        item: state.guide.item,
+        language: state.suite.settings.language,
+    }));
+    const { setView } = useActions({
+        setView: guideActions.setView,
+    });
+
+    if (!item || item.type === GuideType.ARTICLE) {
+        return null;
+    }
+
+    const title = getItemTitle(item, language);
+
+    const hasArticles = item.children.length && item.children[0].type === GuideType.ARTICLE;
+    const hasCategories = item.children.length && item.children[0].type === GuideType.CATEGORY;
+
+    return (
+        <ViewWrapper>
+            <Header back={() => setView('GUIDE_DEFAULT')} label={title} />
+            <Content>
+                {hasArticles && (
+                    <Section>
+                        <SectionHeading>
+                            <Translation id="TR_GUIDE_ARTICLES" />
+                        </SectionHeading>
+                        <Items>
+                            {item.children.map(i => (
+                                <GuideSelectionItem item={i} />
+                            ))}
+                        </Items>
+                    </Section>
+                )}
+                {hasCategories &&
+                    item.children.map(category => (
+                        <Section>
```

Comment:
also missing key param?
```suggestion
                        <Section key={category.id}>
```

Reply:
Resolved.